giraffe/src/utils/fromFlux.ts

```diff
@@ -219,6 +364,49 @@ const splitChunks = (fluxCSV: string): string[] => {
   return chunks
 }
 
+/*
+  A Flux CSV response can contain multiple CSV files each joined by a newline.
+  This function splits up a CSV response into these individual CSV files.
+  See https://github.com/influxdata/flux/blob/master/docs/SPEC.md#multiple-tables.
+*/
+const splitChunksFast = (fluxCSV: string): string[] => {
+  const trimmedResponse = fluxCSV.trim()
```

Comment:
duplicating data here

Reply:
Is there a way to have `trim` apply changes to the string without needing to be assigned onto a new string?

Reply:
instead of starting your index hunting while loop at `0`, start it at `n`, where `n` is determined by finding the first `\S` (non-whitespace character) instance